src/main/java/com/wise23/chariteed/model/Role.java

```diff
@@ -0,0 +1,17 @@
+package com.wise23.chariteed.model;
+
+public enum Role {
+    USER("User"),
+    //    MODERATOR("Moderator"),
+    ADMIN("Admin");
+    private final String value;
```

Comment:
what is the purpose of value here?

Reply:
I changed the class to an enum with the two values `USER` and `ADMIN`

Reply:
@JannesVolkens i think we need 3 roles: Patient, Doctor and Admin.

Reply:
Done